src/js/wrapper.ts

```diff
@@ -109,7 +104,7 @@ export const NATIVE: SentryNativeWrapper = {
         bytesPayload = [...itemPayload];
       } else {
         bytesPayload = utf8ToBytes(JSON.stringify(itemPayload));
-        isFatalCrash = 'level' in itemPayload && itemPayload.level === 'fatal';
+        hardCrashed = isHardCrash(itemPayload) || hardCrashed;
```

Comment:
Instead of doing `|| hardCrashed` I'd rather not check the next events if `hardCrashed = true` so you spare the computation of the next items when at least 1 is `hardCrashed=true`.

Reply:
So is it okay to discard the item after the hardCrash? 

In other word stop processing after first hard crash and send it to the native layer?